lib/cocoa/helpers.rb

```diff
@@ -82,9 +83,9 @@ class NSObject
     attr_reader :object
 
     def self.attach_singular_method method,*__params
-      return if method == :superclass
-      return if method == :class
-      return if method == :new
+      return if method == :superclass ||
+                method == :class ||
+                method == :new
```

Comment:
i thought of that but realised that creating an array and checking if something is in it or not will be slower